src/modules/Dashboard/features/Applet/Schedule/EventForm/EventForm.utils.tsx

```diff
@@ -14,19 +16,131 @@ import {
   TimerType,
 } from 'modules/Dashboard/api';
 import { CalendarEvent } from 'modules/Dashboard/state';
+import { getIsRequiredValidateMessage } from 'shared/utils';
 
 import { removeSecondsFromTime } from '../Schedule.utils';
+import { AvailabilityTab } from './AvailabilityTab';
+import { NotificationsTab } from './NotificationsTab';
+import { TimersTab } from './TimersTab';
 import {
   DEFAULT_END_TIME,
   DEFAULT_IDLE_TIME,
   DEFAULT_START_TIME,
   DEFAULT_TIMER_DURATION,
   SECONDS_TO_MILLISECONDS_MULTIPLIER,
 } from './EventForm.const';
-import { EventFormValues, SecondsManipulation } from './EventForm.types';
+import {
+  EventFormValues,
+  NotificationTimeTestContext,
+  SecondsManipulation,
+} from './EventForm.types';
 
 const { t } = i18n;
 
+export const getEventFormTabs = ({
+  hasAvailabilityErrors,
+  hasTimerErrors,
+  hasNotificationsErrors,
+}: Record<string, boolean>) => [
+  {
+    labelKey: 'availability',
+    content: <AvailabilityTab />,
+    hasError: hasAvailabilityErrors,
+  },
+  {
+    labelKey: 'timers',
+    content: <TimersTab />,
+    hasError: hasTimerErrors,
+  },
+  {
+    labelKey: 'notifications',
+    content: <NotificationsTab />,
+    hasError: hasNotificationsErrors,
+  },
+];
+
+const getStartEndComparisonResult = (startTime: string, endTime: string) => {
+  const startDate = new Date(`2000-01-01T${startTime}:00`);
+  const endDate = new Date(`2000-01-01T${endTime}:00`);
+
+  return startDate < endDate;
```

Comment:
are you sure about format 'hh:mm'? if '1:30' is being compared instead of '01:30', so '1:30' < '12:30' === false

Reply:
yes, the value has additional zero before the number